BoxContentSDK/BoxContentSDK/Models/BOXRepresentation.m

```diff
@@ -87,4 +87,41 @@ - (instancetype)initWithJSON:(NSDictionary *)JSONResponse
     return self;
 }
 
+- (NSDictionary *)composeJSONData
+{
+    NSMutableDictionary *JSONData = [NSMutableDictionary new];
+    if (self.type) {
+        JSONData[BOXAPIObjectKeyRepresentation] = self.type;
+    }
+    if (self.dimensions) {
+        JSONData[BOXAPIObjectKeyProperties] = @{BOXAPIObjectKeyDimensions: self.dimensions};
+    }
+    if (self.status || self.statusCode) {
+
+        NSMutableDictionary *statusDict = [NSMutableDictionary new];
+        if (self.status) {
+            statusDict[BOXAPIObjectKeyState] = self.status;
```

Comment:
I don't think we need any of these conditionals, since `x[y] = nil` is okay.

Reply:
done